### DISCUSSION TITLE
`Promise(never, <Thing>)`, hmm, why not `Always(<Thing>)`?

### DISCUSSION THREAD

**Discussion by nilslindemann** [mint-lang/mint#415] [Language Development]
I am still not fluid in these Promise/Maybe/Result things in functional programming, and so my request may be stupid, but could we have a type `Always(Thing)` which is an alias for `Promise(never, <Thing>)`? Looks like a valid shortcut to me.

**Comment by gdotdesign** [comment]
`Promise` now only has one parameter, so this discussion is not relevant anymore.

**Reply by nilslindemann** [reply]
Oh, ok, it has been a while. Thanks for answering.
